apps/wing/src/wingc.ts

```diff
@@ -93,6 +93,21 @@ export async function load(options: WingCompilerLoadOptions) {
     }
   }
 
+  // preopen all relative paths
+  let iDots = 1;
+  while(iDots < 100) {
+    const dots = "../".repeat(iDots).slice(0, -1);
+    const resolved = resolve(dots);
+    iDots += 1;
+    if (existsSync(resolved)) {
+      preopens[dots] = resolved;
+    }
+    
+    if (resolved === "/" || resolved.match(/^[A-Z]:\\$/i)) {
+      break;
+    }
+  }
```

Comment:
Since we are now preopening all `node_modules` paths [here](https://github.com/winglang/wing/pull/2136/files#diff-e51e2fcb0abcddbff1739190a153f55fa3e991cae1a7e64020de3235c862f180), I don't think this is needed anymore.

Reply:
Those are for the cwd and current node process, which isn't necessarily the same as node_modules that a wing file might try to resolve. Either way, this change didn't really work either so I had to do another approach.

I'm really thinking the final solution for this is going to be calling `require.resolve` from rust itself